lib/runner/VirtualAlexaInvoker.js

```diff
@@ -57,6 +57,15 @@ module.exports = class VirtualAlexaInvoker extends Invoker {
         }
     }
 
+    async invokeBatch(interactions) {
+        const responses = [];
+        for (const interaction of interactions) {
+            const response = await this.invoke(interaction);
```

Comment:
this will wait sequentially for each separate invoke, a Promise.all for an array of promises makes more sense unless we consider memory an issue.

Reply:
The invocations need to be done in sequence, so promise.all will not work.